File: ESCHOOLING.Shared/Models/ChatMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ESCHOOLING.Shared.Models
{
    public class ChatMessage
    {
        public long Id { get; set; }
        public long FromUserId { get; set; }
        public long ToUserId { get; set; }
        public DateTime TimeSent { get; set; }
        public bool HasRecieverRed { get; set; }
        public string MessageText { get; set; }
    }
}

File: ESCHOOLING.Web/wwwroot/js/ChatMessage.js
class ChatMessage {
    constructor(id, fromUserId, toUserId, message, timeSent, hasRead) {
        this.Id = id;
        this.FromUserId = fromUserId;
        this.ToUserId = toUserId;
        this.Message = message;
        this.TimeSent = timeSent;
        this.hasRead = hasRead;

    }
}